examples/inverse/psf_ctf_vertices.py

```diff
@@ -51,21 +46,16 @@
 inverse_operator = mne.minimum_norm.make_inverse_operator(
     info=evoked.info, forward=forward, noise_cov=noise_cov, loose=0.,
     depth=None)
-inverse_operator_vol = mne.minimum_norm.make_inverse_operator(
-    info=evoked.info, forward=forward_vol, noise_cov=noise_cov)
 
 # regularisation parameter
 snr = 3.0
 lambda2 = 1.0 / snr ** 2
+method = 'MNE'  # can be 'MNE' or 'sLORETA'
```

Comment:
why is this back? `method` variable isn't used; I thought @olafhauk removed this line last week.

Reply:
Sorry I just did a `git checkout` that was probably wrong, will fix...